### PR TITLE
Upgrade Cypress to the latest stable version

### PR DESCRIPTION
**Pull request by alfonsomthd**
This also will make easier to set up the future unit/component testing.

Signed-off-by: Alfonso Martínez <almartin@redhat.com>

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -3,9 +3,11 @@ dist
...
 .eslintcache
 cypress/cypress-a11y-report.json
+gui-test-screenshots
 plugins/odf/package.json
 plugins/odf/dist
 plugins/odf/.cache-loader
 plugins/mco/package.json
 plugins/mco/dist
 plugins/mco/.cache-loader
+yarn-error.log
```
Comment (SanjalKatiyar) [inline .gitignore:13]: plz remove extra newline !

File: cypress/cypress.config.ts [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+  video: true,
+  videosFolder: 'gui-test-screenshots/videos/',
+  viewportHeight: 1080,
+  viewportWidth: 1920,
+  e2e: {
+    setupNodeEvents,
+    specPattern: 'cypress/tests/**/*.ts',
+    supportFile: 'cypress/support.ts',
+  },
+});
```
Comment (SanjalKatiyar) [inline cypress/cypress.config.ts:26]: we don't need " "pluginsFile": "cypress/plugin.js" " ??

File: cypress/cypress.json [hunk 3]
```diff
@@ -1,21 +0,0 @@
...
-  "screenshotsFolder": "gui-test-screenshots/screenshots/",
-  "videosFolder": "gui-test-screenshots/videos/",
-  "video": true,
-  "reporter": "node_modules/cypress-multi-reporters",
-  "reporterOptions": {
-    "configFile": "cypress/reporter-config.json"
-  },
-  "screenshotOnRunFailure": true,
-  "supportFile": "cypress/support.ts",
-  "pluginsFile": "cypress/plugin.js",
```
Comment (SanjalKatiyar) [inline cypress/cypress.json:14]: we don't need this in "cypress.config.ts" ??

File: .gitignore (hunk 1 above)
Comment (alfonsomthd) [reply .gitignore:13]: You mean the reverse: to add an extra empty line at the end, right?

File: cypress/cypress.config.ts (hunk 2 above)
Comment (alfonsomthd) [reply cypress/cypress.config.ts:26]: [That option does not exist anymore](https://docs.cypress.io/guides/references/migration-guide#pluginsFile): it's replaced by `setupNodeEvents`.

File: .gitignore (hunk 1 above)
Comment (SanjalKatiyar) [reply .gitignore:13]: sorry !!
yeah, need to add new line :P

File: cypress/cypress.json (hunk 3 above)
Comment (SanjalKatiyar) [reply cypress/cypress.json:14]: okay... missed the import part on top of "cypress.config.ts" file... looks we are good !

Comment (alfonsomthd) [reply cypress/cypress.json:14]: `pluginsFile`: already answered [here](https://github.com/red-hat-storage/odf-console/pull/511#discussion_r1005306558).
The other 2 options are still there.

File: .gitignore (hunk 1 above)
Comment (alfonsomthd) [reply .gitignore]: Done.

**Pull request merged**